source/features/index.tsx

```diff
@@ -104,12 +105,81 @@ const globalReady: Promise<RGHOptions> = new Promise(async resolve => {
 		document.head.append(<style>{options.customCSS}</style>);
 	}
 
+	const testedFeatures: [string[], string[]] | false | undefined = await cache.get('bisect');
+	if (testedFeatures !== undefined) {
+		if (testedFeatures === false) {
+			createMessageBox('Every feature has been disabled. If you still see the bug, try disabling the whole extension.', false);
+			return;
+		}
+
+		const [firstHalf] = testedFeatures;
+		options = {
+			...options,
+			...Object.fromEntries(__features__.map(feature => [`feature:${feature}`, firstHalf.includes(feature)]))
+		};
+
+		if (firstHalf.length === 0) {
+			createMessageBox('Every feature has been disabled. Can you see the bug?');
+			return;
+		}
+
+		if (firstHalf.length === 1) {
+			createMessageBox(<span>The bug is caused by <a href={'https://github.com/sindresorhus/refined-github/blob/main/source/features/' + firstHalf[0] + '.tsx'}><code>{firstHalf[0]}</code></a>.</span>, false);
+			return;
+		}
+
+		createMessageBox(`${Math.ceil(Math.log2(firstHalf.length))} steps remaining. Can you see the bug?`);
+	}
+
 	// Create logging function
 	log = options.logging ? console.log : () => {/* No logging */};
 
 	resolve(options);
 });
 
+function createMessageBox(message: string | Element, yesNoButtons = true): void {
+	delegate(document, '.rgh-bisect-button-choice', 'click', onBisectButtonChoiceClick);
+	delegate(document, '.rgh-bisect-button-end', 'click', onBisectButtonEndClick);
```

Comment:
Had to go with the `delegate` route because `onClick` was giving me serious type headaches. (It demands a `MouseEventHandler<HTMLButton>` and I couldn't fix it with typecasting).

Reply:
If you encounter type issues you can let us fix them, as long as the underlying code actually works.

Reply:
Let's use onClick